website/docs/components/tabs/partials/code/component-api.md

```diff
@@ -9,7 +9,17 @@ The Tabs component is composed of different parts, with their own APIs:
 ### Tabs API
 
 <Doc::ComponentApi as |C|>
-  <C.Property @name="onClickTab" @type="function"/>
+  <C.Property @name="onClickTab" @type="function">
+    Callback function invoked when the one of the tabs is clicked, if provided. The function receives the DOM `event` and the tab's `index` (integer number) as arguments.
+  </C.Property>
+  <C.Property @name="selectedTabIndex" @type="integer">
+    This argument can be used to select the initial tab and control the component state bypassing its default internal state (usually via query parameters).
+    <br/>
+    _Notice: when the initial tab is set using this parameter instead of using the `@isSelected` argument on the `Tab` sub-component, the consumer will need to take care of updating the index whenever the user clicks on one of the tabs (via `onClickTab`)._
```

Comment:
```suggestion
    _Notice: when the initial tab is set using this parameter instead of using the `@isSelected` argument on the `Tab` sub-component, the consumer is responsible for updating the index whenever the user clicks on one of the tabs (via `onClickTab`)._
```
[Suggestion]
This _might_ read a bit better if I understand what you're trying to say.